Tests/SWBBuildSystemTests/BuildCommandTests.swift

```diff
@@ -13,14 +13,16 @@
 import SWBCore
 import SWBProtocol
 import SWBTestSupport
+import SwiftBuildTestSupport
+import SwiftBuild
```

Comment:
This second import adds a dependency that's not in the Package.swift/CMake - is it strictly needed?